src/Sarif.Validation/ResultLogBuilder.cs

```diff
@@ -6,11 +6,12 @@
 using Microsoft.CodeAnalysis.Sarif.Driver;
 using Microsoft.CodeAnalysis.Sarif.Driver.Sdk;
 
-namespace Microsoft.CodeAnalysis.Sarif.SarifValidator
+namespace Microsoft.CodeAnalysis.Sarif.Validation
 {
-    internal class ResultLogBuilder: IDisposable
+    public class ResultLogBuilder: IDisposable
```

Comment:
Since this class is now used both by the SarifValidator tool and by the functional tests, and since it doesn't live _in_ the tool project any more, it must be public.